### PR TITLE
Fixing validation for volume size

### PR DESCRIPTION
**Pull request by hub-cap**

### CODE REVIEW

File: reddwarf/common/exception.py [hunk 1]
```diff
@@ -118,3 +118,8 @@ class VolumeAttachmentsNotFound(NotFound):
 class VolumeCreationFailure(ReddwarfError):
 
     message = _("Failed to create a volume in Nova.")
+
+
+class BadValue(ReddwarfError):
```
Comment (rnirmal) [inline reddwarf/common/exception.py]: We should probably just keep that as BadRequest

Comment (hub-cap) [inline reddwarf/common/exception.py]: It translates to a BadRequest 400 in the service exception map. I just wanted to make it less confusing

**Comment by cp16net** [comment]
+1

**Comment by TimSimpsonR** [comment]
+1

**Comment by rnirmal** [comment]
+1

**Comment by ed-** [comment]
+1

**Comment by imsplitbit** [comment]
+1

**Pull request merged**